eng/pipelines/templates/jobs/archetype-sdk-data.yml

```diff
@@ -0,0 +1,222 @@
+parameters:
+  SDKType: not-specified # Set a default that breaks in obvious ways.
+  ServiceDirectory: not-specified # Set a default that breaks in obvious ways.
+  PreTestSteps: []
+  TestOptions: '$(DefaultOptions)'
+  TestGoals: verify
+  TestMatrix:
+    Linux - Java 8:
+      OSName: 'Linux'
+      OSVmImage: 'ubuntu-16.04'
+      JavaVersion: '1.8'
+      RunTitle: 'Linux on Java 1.8'
+    macOS - Java 8:
+      OSName: 'macOS'
+      OSVmImage: 'macOS-10.13'
+      JavaVersion: '1.8'
+      RunTitle: 'macOS on Java 1.8'
+    Windows - Java 8:
+      OSName: 'Windows'
+      OSVmImage: 'windows-2019'
+      JavaVersion: '1.8'
+      RunTitle: 'Windows on Java 1.8'
+    Linux - Java 11:
+      OSName: 'Linux'
+      OSVmImage: 'ubuntu-16.04'
+      JavaVersion: '1.11'
+      RunTitle: 'Linux on Java 1.11'
+    macOS - Java 11:
+      OSName: 'macOS'
+      OSVmImage: 'macOS-10.13'
+      JavaVersion: '1.11'
+      RunTitle: 'macOS on Java 1.11'
+    Windows - Java 11:
+      OSName: 'Windows'
+      OSVmImage: 'windows-2019'
+      JavaVersion: '1.11'
+      RunTitle: 'Windows on Java 1.11'
+
```

Comment:
Data track doesn't have the same interdependencies as client track and the **from source** runs don't really make much sense.

Reply:
OK we can add it later if we find a gap we need to fill.

Reply:
The from source builds were never intended to run for anything other than client. Part of this change removed the [conditions](https://github.com/Azure/azure-sdk-for-java/pull/7571/files#diff-a36606b54a95a0d62cfd379adb42c11eL239) in the archetype-sdk-client that were limiting this run.